### ISSUE TITLE
Not pulling from API

### ISSUE THREAD

**Issue by MayhemParadox** [delightedCrow/WallberryTheme#4]
My board is working fine and looks great, it was pulling images and working properly up until a few days ago. Now it is not pulling images from my unsplash API. Everything is showing operational and that I have 50/50 pulls remaining for the hour. Any suggestions?

**Comment by scuppasteve** [comment]
I have the exact same issue. I thought maybe i was out of API pulls, but it seems like i am making the pulls, but nothing is displayed for the background. Everything else is working.

**Comment by Ladinn** [comment]
Same issue here. The Unsplash API portal shows that requests are being made, but nothing is displayed on the screen. Just like @MayhemParadox, everything was working until a few days ago.

**Comment by delightedCrow** [comment]
interesting - does anyone have any errors showing up in the console log? Also, is the issue intermittent, or does it not pull images at all anymore (even after a restart)?

**Comment by scuppasteve** [comment]
No images at all anymore, i have restarted the software and the whole OS. its just black now. Where is the console log so i can grab it for you.

**Comment by Ladinn** [comment]
@scuppasteve If you're using PM2 on a Pi, it's /home/pi/.pm2/logs/MagicMirror-out.log.

@delightedCrow I've got no errors in the error log and the module appears to load without issues- the only console output is:

`No helper found for module: WallberryTheme.
`

**Comment by delightedCrow** [comment]
you can also start MM2 in developer mode by running `npm start dev` from your magic mirror directory - it should start MM2 with the chromium dev console open so you can see if there are any network requests erroring out

**Comment by Ladinn** [comment]
@delightedCrow Got this in Chromium:

`GET https://api.unsplash.com/photos/random?client_id=f862aba00da77818cee68ff3974385172fea79b796b8067deeccc1a620a7ecd7&collections=2589108,1538150,225,827743,162468,1242150&orientation=portrait&w=461&h=1856 404 ()`

**Comment by delightedCrow** [comment]
I'm having trouble recreating the error, but I'm wondering if it might be an issue with Unsplash having an issue finding a photo in a certain collection and returning a 404. @ladinn do you get the same error when you leave the `collections` parameter blank in your config?

**Comment by Ladinn** [comment]
No, here's my config.js- the same ones from collections query in the URI:

`collections: "2589108,1538150,225,827743,162468,1242150"`

**Comment by delightedCrow** [comment]
awesome thanks. I can reproduce the issue now. For some reason Unsplash seems to be erroring out and returning a 404 whenever there is more than one collection specified. For example, using the following in the config seems to make things work again for me:
`collections: ""`
`collections: "225"`

This definitely seems to be sudden new behavior for Unsplash's API, as multiple collections have been working before now :( 
Paring down to one collection seems to fix the issue in the short term but I'm gonna keep investigating.

**Comment by scuppasteve** [comment]
I have verified this restores background, with the limitation you stated.

**Comment by scuppasteve** [comment]
Thank you for the help.

**Comment by delightedCrow** [comment]
I've submitted the bug to Unsplash. Looks like they've been having similar API bugs in the last couple days with some requests returning 404s and they've said they're working on those so hopefully they'll get things back to normal soon. Fingers crossed :)
